package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -180,6 +180,10 @@ If you use the Mean Squared Displacement analysis code in
 
 .. _`10.21105/joss.00877`: https://doi.org/10.21105/joss.00877
 
+For additional information on non-gaussian parameter, consider [Fuchs1998]_.
```

Comment:
What is this in relation to exactly? Is there a specific thing in H-bonds that you are referring to?

Reply:
Nope I think I was on the wrong branch, this is foreshadowing.